pkg/imagehandler/imagehandler.go

```diff
@@ -17,12 +17,59 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"os"
+	"path"
+	"regexp"
 	"sync"
 
 	"github.com/go-logr/logr"
 	"github.com/google/uuid"
+
+	"github.com/openshift/image-customization-controller/pkg/env"
+)
+
+const (
+	hostArchitectureKey = "host"
 )
 
+var deployImagePattern = regexp.MustCompile(`ironic-python-agent_(\w+)\.(iso|initramfs)`)
```

Comment:
Ideally this wouldn't be hard-coded either (could be a separate env var, or calculate it from the DeployIso/DeployInitrd).